common/diplomatic_plays/dvg_diplomatic_plays.txt

```diff
@@ -548,7 +548,7 @@ dp_unify_persia = {
 		NOT = { has_modifier = diplomatic_plays_blocked }
 		NOT = { is_country_type = decentralized }
 		is_subject = no
-		#has_technology_researched = nationalism
+		has_technology_researched = nationalism
```

Comment:
Any reason to add this again? I removed this because Persia otherwise acted up with weird borders most of the time.
Unless we setup better AI strategies/state priorities.

Reply:
Mainly for balance, because thing is most of the time Karlani Shahdom/Afghanistan just steamrolls the Persian minors early in the game. With a tech requirement, it could give players some time to prepare or sort out their country's situation before a unification war.